network/hermes/restore-keys.sh

```diff
@@ -6,9 +6,16 @@ set -e
 
 ### Sleep is needed otherwise the relayer crashes when trying to init
 sleep 1
+
 ### Restore Keys
-$HERMES_BINARY -c ./network/hermes/config.toml keys restore test-1 -m "alley afraid soup fall idea toss can goose become valve initial strong forward bright dish figure check leopard decide warfare hub unusual join cart"
+echo "alley afraid soup fall idea toss can goose become valve initial strong forward bright dish figure check leopard decide warfare hub unusual join cart" > ./test_1_m.txt
+$HERMES_BINARY --config ./network/hermes/config.toml keys delete --chain test-1 --key-name testkey_1
+$HERMES_BINARY --config ./network/hermes/config.toml keys add --key-name testkey_1 --chain test-1 --mnemonic-file ./test_1_m.txt
+rm ./test_1_m.txt
```

Comment:
Since this is bash, you can use process substitution syntax and skip temporary file creation, see example:
```bash
cat <(echo "this string was read from fake file")
```